quickwit/quickwit-doc-mapper/src/default_doc_mapper/date_time_format.rs

```diff
@@ -323,4 +351,32 @@ mod tests {
         ];
         assert_eq!(date_time_formats, &expected_date_time_formats);
     }
+
+    #[test]
+    fn test_fail_date_time_input_format_from_str_with_unknown_format() {
+        let formats = vec![
+            "test%",
+            "test-%v",
+            "test-%q",
+            "unix_timestamp_secs",
+            "unix_timestamp_seconds",
+        ];
+        for format in formats {
+            let error_str = DateTimeInputFormat::from_str(format)
+                .unwrap_err()
+                .to_string();
+            assert_eq!(error_str, format!("Unknown input format: `{format}`."));
+        }
+    }
+
+    #[test]
+    fn test_fail_date_time_output_format_from_str_with_unknown_format() {
```

Comment:
there is one redundant test I think.

Reply:
Sorry, I don't understand. 
There is the DateTimeInputFormat and DateTimeOutputFormat with a different error message for this purpose.
Also DateTimeInputFormat has the   `unix_timestamp_*` set of formats

Do you mean only testing one is enough?

Reply:
My mistake, sorry for the bad comment!

Reply:
No worries :+1: